src/System.Security.Cryptography.X509Certificates/tests/TestData.cs

```diff
@@ -1470,5 +1470,107 @@ internal struct ECDsaCngKeyValues
             public byte[] QY;
             public byte[] D;
         }
+
+        internal static readonly RSAParameters RsaBigExponentParams = new RSAParameters
```

Comment:
Yep, but that file isn't currently included in this project, and it feels like unnecessary contamination to include it here.